src/server/set_family_test.cc

```diff
@@ -412,4 +412,17 @@ TEST_F(SetFamilyTest, SAddEx) {
   EXPECT_THAT(Run({"saddex", "key", "KEEPTTL", "2"}), ErrArg("wrong number of arguments"));
 }
 
+TEST_F(SetFamilyTest, CheckSetLinkExpiryTransfer) {
```

Comment:
can you please add an equivalent test in `src/core/string_set_test.cc`, @mkaruza ?

Reply:
Added